source/generate_uids/generate_storage_sopclass_uids.py

```diff
@@ -66,11 +66,11 @@ def replace_bad_characters(name):
     return name.translate(translate_table)
 
 
-def uid_line(uid, name):
+def uid_line(uid, keyword):
     """Return the UID class definition line to be written into the python file.
     """
     # add a line break after UID to avoid too long lines
-    return "{} = UID(\n    '{}')\n".format(sop_class_name(name), uid)
+    return f"{keyword} = UID(\n    '{uid}')\n"
 
```

Comment:
Might as well remove `sop_class_name()` function - I don't it removed in the diff.

Side note: man, the f-strings make this so much more readable.

Reply:
I'll leave it, the whole file is obsolete since we can build all those UID constants dynamically now